### PR TITLE
Standardizing/Proofreading Card Effects

### PR DESCRIPTION
**Pull request by Llendlar**

### CODE REVIEW

**Comment by BeligerentBanana** [comment]
Don't submit large commits, makes it hard to review.

File: cardlibrary/baselibrary.lua [hunk 1]
```diff
@@ -482,7 +482,7 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Cost"] = {["Yellow"] = 1,["Neutral"] = 2,},
 		["Effect"] = {
 			Name = "Yarrge",
-			Description = "Gain 1 Yellow Icon. This card cannot generate icons.",
+			Description = "Can't generate icons. Gain 1 yellow icon.",
```
Comment (Espithel) [inline cardlibrary/baselibrary.lua]: Can't generate icons naturally*

File: cardlibrary/baselibrary.lua [hunk 2]
```diff
@@ -554,7 +554,7 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Cost"] = {["Neutral"] = 2,},
 		["Effect"] = {
 			Name = "Revival",
-			Description = "Generate a Targeting Blip. Targeting Blips are used to trigger Target Effects.",
+			Description = "Add a Targeting Blip to your hand. Targeting Blips are used to trigger target effects.",
```
Comment (Espithel) [inline cardlibrary/baselibrary.lua]: Put

File: cardlibrary/baselibrary.lua [hunk 3]
```diff
@@ -849,7 +849,7 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Cost"] = {["Neutral"] = 1,["Red"] = 3,},
 		["Effect"] = {
 			Name = "seventh jutsu of the final sun's rend",
-			Description = "Deal 400 damage to a target fighter. You lose 600 life.",
+			Description = "Deal 400 damage to a target fighter. Lose 600 life.",
```
Comment (Espithel) [inline cardlibrary/baselibrary.lua]: You need to state what loses life.

File: cardlibrary/baselibrary.lua [hunk 4]
```diff
@@ -1488,7 +1488,7 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Bio"] = "My boy, bombs are what all true warriors strive for!",
 		["Effect"] = {
 			Name = "DO0M",
-			Description = "Deal 500 to both players, then draw 1 card.",
+			Description = "Both players lose 500 life. Draw 1 card.",
```
Comment (Espithel) [inline cardlibrary/baselibrary.lua]: Who draws the card?

File: cardlibrary/baselibrary.lua [hunk 5]
```diff
@@ -1514,9 +1514,9 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Bio"] = "Some people wish to win the Bloxxies, but there can only be one.",
 		["Effect"] = {
 			Name = "Bloxy Victory",
-			Description = "Increase the Attack of all your fighters by 600, then decrease their health by 300.",
+			Description = "Deal 300 damage to allied fighters. Increase the power of allied fighters by 600.",
```
Comment (Espithel) [inline cardlibrary/baselibrary.lua]: All allied fighters, not just allied fighters.

File: cardlibrary/baselibrary.lua [hunk 6]
```diff
@@ -1960,7 +1960,7 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Cost"] = {["Green"] = 1,["Neutral"] = 1,},
 		["Effect"] = {
 			Name = "Render",
-			Description = "Whenever BadBreadMaker is targeted by a creature, that creature and BadBreadMaker gain 300 health.",
+			Description = "Whenever BadBreadMaker is targeted by a fighter, increase the health of it and BadBreadMaker by 300.",
```
Comment (Espithel) [inline cardlibrary/baselibrary.lua]: increase the health of that fighter and badbreadmaker by 300.
We need to get rid of ambiguity.

File: cardlibrary/baselibrary.lua [hunk 7]
```diff
@@ -2050,7 +2050,7 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 				["Id"] = 655012667,
 				["Effect"] = {
 					Name = "Vampire",
-					Description = "Whenever you gain life, put a Love Letter into your hand.",
+					Description = "Whenever you gain life, add a Love Letter to your hand.",
```
Comment (Espithel) [inline cardlibrary/baselibrary.lua]: Put, not add.

File: cardlibrary/baselibrary.lua [hunk 8]
```diff
@@ -2065,7 +2065,7 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Cost"] = {["Neutral"] = 3,},
 		["Effect"] = {
 			Name = "Vampire",
-			Description = "Whenever you gain life, put a kill script into your hand.",
+			Description = "Whenever you gain life, add a Kill Script to your hand.",
```
Comment (Espithel) [inline cardlibrary/baselibrary.lua]: Put, not add.

File: cardlibrary/baselibrary.lua [hunk 9]
```diff
@@ -2296,7 +2296,7 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Cost"] = {["Neutral"] = 1,["Blue"] = 1,},
 		["Effect"] = {
 			Name = "Amp",
-			Description = "Whenever this fighter is targeted, put a random Common card to your hand.",
+			Description = "Whenever this fighter is targeted, add a random common card to your hand.",
```
Comment (Espithel) [inline cardlibrary/baselibrary.lua]: Put, not add.

**Pull request closed without merging**